KILTprotocol/sdk-js
# chore: outsourced constructor input checks

## fixes https://github.com/KILTprotocol/ticket/issues/154


## How to test:

Try to build any constructor refactored instance with unprovided data.
Should throw error, and not create an instance.
## Checklist:
